packages/web-components/fast-components-msft/src/text-area/fixtures/text-area.html

```diff
@@ -61,12 +61,20 @@ <h5>Disabled</h5>
     <fast-text-area appearance="filled" disabled>
         <span>label</span>
     </fast-text-area>
-    <fast-text-area appearance="filled" disabled placeholder="placeholder"></fast-text-area>
+    <fast-text-area
+        appearance="filled"
+        disabled
+        placeholder="placeholder"
+    ></fast-text-area>
 
     <!-- Read only -->
     <h5>Read only</h5>
-    <fast-text-area appearance="filled" value="Readonly text area" readonly>label</fast-text-area>
-    <fast-text-area appearance="filled" value="Readonly text area" readonly>label</fast-text-area>
+    <fast-text-area appearance="filled" value="Readonly text area" readonly
+        >label</fast-text-area
```

Comment:
o.0

Reply:
Oh, that is interesting... i'll see if i can change some settings